integration/assets_test.go

```diff
@@ -1,17 +1,17 @@
 //go:build integration
 
 // Package integration holds the project's integration tests.
-//
-// This file tests the assets are available as expected.
 package integration
 
+// This file tests the assets are available as expected.
```

Comment:
Is it possible to put the "This file..." docs above the package documentation in its own comment block? Or, I'm wondering whether it's useful. I think before it was distinguishing between package and file (but incorrectly getting included in the package docs), but I'm less clear about its usefulness if it's lower down in the file (above the imports).

This would apply to all these.